lib/animation.js

```diff
@@ -210,6 +226,65 @@ Animation.prototype.speed = function(speed) {
 };
 
 /**
+ * This function is called in each frame of our animation
+ * Users need not call this. It's automatic
+ */
+
+ Animation.prototype.loopFunction = function(loop) {
+
+  // Find the current timeline progress
+  var progress = this.calculateProgress(loop.calledAt);
+
+  // Find the left and right cuePoints/keyFrames;
+  var indices = this.findIndices(progress);
+
+  // Get tweened value
+  var val = this.tweenedValue(indices, progress);
+
+  // call render function
+  this.target[Animation.render](val);
+
+  // If this animation has been running for more than 5 seconds
+  if (loop.calledAt > this.fallBackTime) {
+    this.fallBackTime = Infinity;
+    this.playLoop.stop();
```

Comment:
There are other places where `this.playLoop` is checked before stop is called, should it be same here, or is this known to be safe?